src/core/less/_defaults/icons.less

```diff
@@ -0,0 +1,134 @@
+// --------------------------------------------------
+// Icons
+// --------------------------------------------------
+@font-face {
+  font-family: "vanilla";
+  src: url("fonts/vanilla.eot");
+  src: url("fonts/vanilla.eot?#iefix") format("embedded-opentype"),
```

Comment:
we stopped including the .eot, .ttf & .svg versions of the vanilla font a while back didn't we?

Reply:
Done